atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -165,8 +171,35 @@ def get_default_image_build_conf(self):
             'install_tree': install_tree,
             'repo': repo,
         }
+        config_fp = StringIO(self.DEFAULT_IMAGE_BUILD_CONF.format(**kwargs))
+
+        config = ConfigParser()
+        config.readfp(config_fp)
+
+        self.update_config_from_dockerfile(config)
```

Comment:
You *could* operate on `kwargs` instead of `config`, but I don't have a strong preference.

Reply:
I was thinking about that, but `version` is not in template (has no hardcoded default) and it is required so I cannot just add it into template. Updating ConfigParser directly looked more elegant to me than adding special logic to update default config only when it is defined in Dockerfile.